[Docs] HA Tracker: Add minimum configuration guide. (#2872)
cortexproject/cortex: docs/production/ha-pair-handling.md

### Server Side

The minimal configuration requires:

> * Enabling the HA tracker code path

Reviewer: I would be more specific. I would also omit the "code path"

```suggestion
* Enabling the HA tracker via `-distributor.ha-tracker.enable=true` CLI flag (or its YAML config option)
```

> * Configuring the KV store for the ring (See: [Ring/HA Tracker Store](https://cortexmetrics.io/docs/configuration/arguments/#ringha-tracker-store))

Reviewer: Please use internal links, so:

```suggestion
* Configuring the KV store for the ring (See: [Ring/HA Tracker Store](../configuration/arguments.md#ringha-tracker-store))
```

> * Setting the limits configuration to accept samples

Reviewer: ```suggestion
* Setting the limits configuration to accept samples via `-distributor.ha-tracker.enable-for-all-users` (or its YAML config option)
```

> ```yaml

Reviewer: ```suggestion

The following configuration snippet shows an example of the HA tracker config via YAML config file:

```yaml
```

limits:
  ...
  accept_ha_samples: true
  ...
distributor:
  ...
  ha_tracker:
    enable_ha_tracker: true
    ...
    kvstore:
      [store: <string> | default = "consul"]
      [consul | etcd | memberlist: <config>]
      ...
  ...
```

> For further configuration file documentation, see the [distributor section](https://cortexmetrics.io/docs/configuration/configuration-file/#distributor_config) and [Ring/HA Tracker Store](https://cortexmetrics.io/docs/configuration/arguments/#ringha-tracker-store).

Reviewer: ```suggestion
For further configuration file documentation, see the [distributor section](../configuration/config-file-reference.md#distributor_config) and [Ring/HA Tracker Store](../configuration/arguments.md#ringha-tracker-store).
```
